app/assets/stylesheets/modules/research-starter.scss

```diff
@@ -0,0 +1,72 @@
+.research-starter:before {
+  position: relative;
+  top: 10px;
+  left: 10px;
+  font-size: 16px;
+  font-weight: 400;
+  color: #8c1515;
+  text-transform: uppercase;
+  letter-spacing: 1px;
+  content: "Research Starter";
```

Comment:
@jkeck, there was sr-only text specified separately in the ticket. That, combined with the css content, was the only way I could get the heading structure and the open/close behaviour to behave & look how I wanted.